src/v/cloud_storage/async_manifest_view.cc

```diff
@@ -1281,12 +1348,38 @@ async_manifest_view::hydrate_manifest(
           path,
           manifest.size());
         co_return std::move(manifest);
-    } catch (const ss::gate_closed_exception&) {
-        vlog(_ctxlog.debug, "gate closed while hydrating manifest: {}", path);
+    } catch (const ss::gate_closed_exception& e) {
```

Comment:
maybe use `ssx::is_shotdown_exception` instead?

```
} catch (...) {
   auto ex = std::current_exception();
   if (ssx::is_shutdown_exception(ex)) {
      vlog(_ctxlog.debug, "blah blah {}", ex);
      co_return error_outcome::shutting_down;
   }
}
```

Reply:
Oh. I didn't know about this. Will try.

Reply:
Worked neatly. Used it here and at one other exception handling site.